Client/src/Utils/Theme/constants.js

```diff
@@ -35,6 +38,7 @@ const paletteColors = {
 	gray880: "#0C0C0E",
 	gray890: "#09090B",
 	blueGray20: "#E8F0FE",
+	blueGray50: "#CDE2FF",
```

Comment:
Can you add a print of the element where this color is being applied? For the other colors, please use the ones that already exist, and instead of creating new references, use  an existing reference for that color. The idea is that here we are not aware of where the color is being called (so avoid calling them 'stroke' or `gauge`. We currently have those in the code base, but we are increasingly dropping that approach

Reply:
![image](https://github.com/user-attachments/assets/2339cd47-4d24-4f8c-afd6-e3bac72b1c5e)
On Red arrow grey25: "#FCFCFC",
On Black arrow blueGray50: "#CDE2FF",
On Yellow arrow white200:"#F6F6F6",
(this colors are mentioned in figma design)

if i used color which you mentioned then UI will look like this
![image](https://github.com/user-attachments/assets/90563aee-4227-4172-a2e0-8dc5ff24643c)

Reply:
Can you fix the alignment issue here ?

![398573263-2339cd47-4d24-4f8c-afd6-e3bac72b1c5e](https://github.com/user-attachments/assets/652da6fe-d304-4aa0-b605-203540b8d60a)